modules/install-central-operator-tech-preview.adoc

```diff
@@ -49,14 +49,28 @@ $ oc create secret generic external-db-password \ <1>
 <1> If you use Kubernetes, enter `kubectl` instead of `oc`.
 <2> Replace `password.txt` with the path of the file which has the plain text password.
 . Return to the {product-title} operator page in the {ocp} web console. Under the *Provided APIs* section, select *Central*. Click *Create Central*.
+. Optional: If you are using declarative configuration, next to *Configure via:*, click *YAML view*.
```

Comment:
Just reading through it, it might not be clear to users that they should replace `declarative-configs` and `sensitive-declarative-configs` with the name of their respective config maps / secrets they want to use. Do you think it's worthwhile calling this out somewhere or it's just me assuming something wrong?